src/libraries/System.Net.Sockets/tests/FunctionalTests/DisposedSocketTests.cs

```diff
@@ -752,13 +752,14 @@ public void EndAccept_Throws_ObjectDisposed()
         [InlineData(true)]
         public async Task NonDisposedSocket_SafeHandlesCollected(bool clientAsync)
         {
-            List<WeakReference> handles = await CreateHandlesAsync(clientAsync);
-            RetryHelper.Execute(() =>
+            TimeSpan timeout = TimeSpan.FromSeconds(10);
```

Comment:
TestSettings.PassingTestTimeout instead of 10?

Reply:
Done.

Reply:
I don't see any code update @antonfirsov

Reply:
Somehow forgot to push, now it's there, thanks!